lib/datastore/query.js

```diff
@@ -75,7 +75,15 @@ function Query(namespace, kinds) {
 }
 
 /**
+ * @param {boolean=} autoPaginateVal - Have pagination handled automatically.
+ *     Default: true.
  * @return {module:datastore/query}
+ *
+ * @example
+ * // Retrieve a list of people, but disable pagination
+ * var query = dataset.createQuery('Person')
+ *   .hasAncestor(dataset.key(['Person', 1234]))
+ *   .autoPaginate(false);
```

Comment:
It would be awesome to see a full fledged iterative query example, eg https://github.com/GoogleCloudPlatform/gcloud-node/blob/master/lib/datastore/request.js#L568